tests/e2e/utils/pages/dashboard.js

```diff
@@ -0,0 +1,131 @@
+import MockRequests from '../mock-requests';
+
+/**
+ * Dashboard page object class.
+ */
+export default class DashboardPage extends MockRequests {
+	/**
+	 * @param {import('@playwright/test').Page} page
+	 */
+	constructor( page ) {
+		super( page );
+		this.page = page;
+		this.freeListingRow = this.page.locator(
+			'.gla-all-programs-table-card table tr:nth-child(2)'
+		);
+		this.editFreeListingButton = this.freeListingRow.getByRole( 'button', {
+			name: 'Edit',
+		} );
+	}
+
+	/**
+	 * Close the current page.
+	 */
+	async closePage() {
+		await this.page.close();
+	}
+
+	/**
+	 * Mock all requests related to external accounts such as Merchant Center, Google, etc.
+	 */
+	async mockRequests() {
+		// Mock Reports Programs
+		this.fulfillMCReportProgram( {
```

Comment:
It'd probably be better to add `await` in this call and also other calls below. Otherwise this function `mockRequests` would return void directly without waiting these calls to be finished.